### DISCUSSION TITLE
Phidget VINT with 3/4 wire PT100/PT1000 RTDs Issue

### DISCUSSION THREAD

**Discussion by PKeyes505** [artisan-roaster-scope/artisan#700] [Q&A] [answered]
Not sure If I am doing something wrong or if both ET and BT Channels are reading the value from Channel A under TMP1200 RTD settings.

I started with two 4-wire PT1000 RTDs from Phidget and that original setup worked just fine.

When I upgraded the SR800 with an extension tube, I replace one of the RTDs with a 3-wire PT100 RTD and here are my sensors.  
![image](https://user-images.githubusercontent.com/61755459/134719500-5b3df84b-13db-45d6-a03d-f7c88d3f7e27.png)

**Artisan displays the RTD that matches what ever is configured correctly in Channel A.**  

Channel A (ET) - Correct Setting
     TYPE PT1000 3850
     Wiring 4-Wire
Channel B (BT) - Correct Setting
    TYPE PT100 3920
    Wiring 3-Wire
LCDs shows probe matching channel A and correct assignment (ET)
ET: 19.7
BT: -.-

If I change Channel A to the incorrect settings
Channel A (ET) - Incorrect Setting but LCDs shows probe matching channel A and correct assignment (BT)
     TYPE PT100 3920
     Wiring 3-Wire
Channel B (BT) - Correct Setting
    TYPE PT100 3920
    Wiring 3-Wire
LCDs shows Probe matching Channel A.
ET: -.-
BT: 20.0

If I change to
Channel A (ET)
     TYPE PT100 3920 - Incorrect Setting but LCDs shows probe matching channel A and correct assignment (BT)
     Wiring 3-Wire
Channel B
    TYPE PT1000 3850 - Incorrect Setting
    Wiring 4-Wire
LCDs show 
ET: -.-
BT: 20.0

![image](https://user-images.githubusercontent.com/61755459/134720353-69e5a841-5204-4601-a2a9-3fa64a6f3cb3.png)

**Comment by PKeyes505** [comment]
Should there be a Phidget TMP1200 2xRTD A/B?
![image](https://user-images.githubusercontent.com/61755459/134725800-776016a0-8b20-4618-bcd7-5a92e8978578.png)


If I make them virtual devices then it works correctly.
![image](https://user-images.githubusercontent.com/61755459/134726786-20f8e17e-ab48-49fa-bac8-d6c206213476.png)
![image](https://user-images.githubusercontent.com/61755459/134726820-f6b78838-3a66-4d2c-84c3-5bf1dd89f0b2.png)

**Comment by MAKOMO** [comment, marked as answer]
The idea to have two different Phidget TMP1200 1xRTD settings (one named "TMP1200 1xRTD A" and one named "TMP1200 1xRTD B" is exactly for the case that you have two different types of probes connected to your TMP1200 modules.

You would eg. use "TMP1200 1xRTD A" as main device (note those TMP1200 are one channel devices thus this will only deliver data for BT not ET) and add an extra device of type "TMP1200 1xRTD B" to read your second probe (ET).  Then you map the readings of the extra device via a symbolic formula (Y3) to the second channel of the main device.

This is exactly what the machine setup under menu Config >> Machine >> Phidget >> VINT TMP1200 2xRTD does. However, that setup assumes 4-wires PT1000 connected to both modules. So in your case you would start with that setup and just switch the device type of the extra device channel from "TMP1200 1xRTD A" to "TMP1200 1xRTD B".

**Reply by PKeyes505** [reply]
Thank you for the response! I was documenting as went and will post for more visual folks.
1. You would eg. use "TMP1200 1xRTD A" as main device (note those TMP1200 are one channel devices thus this will only deliver data for BT not ET) 
![image](https://user-images.githubusercontent.com/61755459/134734826-25f0b4c0-d0d5-4c87-873c-84a89a791556.png)
2. add an extra device of type "TMP1200 1xRTD B" to read your second probe (ET). 
![image](https://user-images.githubusercontent.com/61755459/134734622-d8359599-e711-4da4-ace1-7b59a6d91c47.png)
3. Then you map the readings of the extra device via a symbolic formula (Y3) to the second channel of the main device.
![image](https://user-images.githubusercontent.com/61755459/134734694-8a8a39d7-747c-47e8-88fa-d4c51dca3320.png)
